src/plugins/intel_cpu/src/nodes/gather.cpp

```diff
@@ -41,8 +42,76 @@ bool Gather::isSupportedOperation(const std::shared_ptr<const ov::Node>& op, std
     return true;
 }
 
+class GatherShapeInfer : public ShapeInferEmptyPads {
```

Comment:
could you please put all the shape inference related code to an anonymous namespace, so that it is not visible outside this translation unit?

Reply:
Applied. Thanks Maksim!